src/documentation/constants/constants.ts

```diff
@@ -407,3 +407,5 @@ export const CHI_VISIBLE_VERSION = [
   '2.1.0',
   '2.0.0'
 ];
+
+export const SIZES: string[] = ['sm', 'md', 'lg', 'xl'];
```

Comment:
```suggestion
export const sizes: string[] = ['sm', 'md', 'lg', 'xl'];
```

Reply:
In talks with JC we decided to leave the name in uppercase